bluewin.ch

```diff
@@ -1,41 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
-  <emailProvider id="bluewin.ch">
+  <emailProvider id="Swisscom">
```

Comment:
ID must be a domain

Reply:
what is the preferred domain here then? The documentation [ConfigFileFormat](https://wiki.mozilla.org/Thunderbird:Autoconfiguration:ConfigFileFormat) doesn't specify anything specific.
The company domain that bluewin.ch & bluemail.ch belong to would be swisscom.ch though no customer mail is run over this domain.

Reply:
swisscom.ch should be good, then. Per spec, customer domains are under the `<domain>` tags.

Reply:
3969ae7b475ab16eb52e55a2e819eb6551cddacb fixes this.
Is there more we should address?